fe/fe-core/src/main/java/org/apache/doris/nereids/properties/ChildrenPropertiesRegulator.java

```diff
@@ -197,12 +199,53 @@ public Boolean visitPhysicalFilter(PhysicalFilter<? extends Plan> filter, Void c
         return true;
     }
 
-    private boolean isBucketShuffleDownGrade(DistributionSpecHash srcSideSpec) {
-        boolean isBucketShuffleDownGrade = ConnectContext.get().getSessionVariable().isEnableBucketShuffleDownGrade();
+    private boolean isBucketShuffleDownGrade(Plan oneSidePlan, DistributionSpecHash otherSideSpec) {
+        // improper to do bucket shuffle join:
+        // oneSide:
+        //      - base table and tablets' number is small enough (< paraInstanceNum)
+        // otherSide:
+        //      - ShuffleType.EXECUTION_BUCKETED
+        boolean isBucketShuffleDownGrade = ConnectContext.get().getSessionVariable().isEnableBucketShuffleJoin();
```

Comment:
when `enableBucketShuffleJoin = false`, we should never do bucket shuffle at all. so when `isEnableBucketShuffleJoin = false`, function `isBucketShuffleDownGrade` should always return true?

Reply:
forget to change the name, isBucketShuffleDownGrade should be isEnableBucketShuffle.